autogpt_platform/backend/migrations/20250313043016_update_store_submissions_format/migration.sql

```diff
@@ -0,0 +1,326 @@
+/*
+  Warnings:
+
+  - The values [DAFT] on the enum `SubmissionStatus` will be removed. If these variants are still used in the database, this will fail.
+  - You are about to drop the column `isApproved` on the `StoreListing` table. All the data in the column will be lost.
```

Comment:
Can we update the summary here? Some seem to be contradictory.

Reply:
oh yeah sorry I just popped all the versions in one after another